content/blog/pulumi-aws-cloudcontrol-provider/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Announcing the General Availability of Pulumi’s AWS Cloud Control Provider (formerly AWS Native) "
+allow_long_title: true
+date: 2024-04-25T09:00:00-07:00
+draft: false
+meta_desc: >-
+    Pulumi announces the GA of its AWS Cloud Control Provider, offering Day 1 support for new AWS features and enhancing cloud management tools.
+meta_image: 
+authors:
+    - tejitha-raju
+    - matt-jeffryes
+tags:
+    - aws 
+    - provider 
+    - cloudcontrol 
+---
+We're excited to announce the general availability of the new [AWS Cloud Control Provider] for Pulumi, previously AWS Native. AWS is one of the most-used cloud providers across the Pulumi ecosystem, and with this launch, we are focused on delivering Day 1 availability and faster access to new AWS capabilities to all Pulumi users. As part of this launch, we are also renaming the provider from “AWS Native” to "AWS Cloud Control Provider," based on customer feedback during the preview phase, to provide users with a broader range of advanced cloud management tools exposed by the AWS Cloud Control API. This provider includes the following features and benefits, 
```

Comment:
Some possible small wording tweaks (I'm mostly just trying to avoid "with this launch, we are focused on delivering"; as a user I'm not really so interested in launches and "focus" as I am on commitments and capabilities.)

```suggestion
We're excited to announce the general availability of the new [AWS Cloud Control Provider] for Pulumi (previously "AWS Native"). AWS is one of the most-used cloud providers across the Pulumi ecosystem, and we are committed to providing the fastest possible access to new AWS capabilities to all Pulumi users. This provider delivers on that promise by providing native support for all resources in the AWS Cloud Control APIs. As part of this launch, we are also renaming the provider from “AWS Native” to "AWS Cloud Control Provider," based on customer feedback during the preview phase, to provide users with a broader range of advanced cloud management tools exposed by the AWS Cloud Control API. This provider includes the following features and benefits, 
```